2024-01-17 15:58:32,241 - ERROR - Failed to load anomaly detection model: 
Traceback (most recent call last):
  File "/Users/apikorus/model/network_test.py", line 34, in <module>
    model = joblib.load(MODEL_PATH)
            ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/lib/python3.11/site-packages/joblib/numpy_pickle.py", line 650, in load
    with open(filename, 'rb') as f:
         ^^^^^^^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: '/Users/apikorus/model/mymodel.pkl'
